### PR TITLE
Added Cxx14 toolchains for android

### PR DESCRIPTION
**Pull request by belveder79**
cheers...

### CODE REVIEW

**Comment by belveder79** [comment]
added the toolchains to table as requested...

File: android-ndk-r11c-api-16-armeabi-c14.cmake [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+polly_init(
+    "Android NDK ${ANDROID_NDK_VERSION} / \
+API ${CMAKE_SYSTEM_VERSION} / ${CMAKE_ANDROID_ARCH_ABI} / \
+32-bit ARM / c++14 support"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake") # before toolchain!
```
Comment (ruslo) [inline android-ndk-r11c-api-16-armeabi-c14.cmake]: > android-ndk-r11c-api-16-armeabi-c14.cmake

`c14` is not the same as `cxx14`, see for example https://github.com/ruslo/polly/blob/master/gcc-5-cxx14-c11.cmake

The name of this toolchain should be `android-ndk-r11c-api-16-armeabi-cxx14.cmake`

**Comment by belveder79** [comment]
sorry, I was sloppy... I fixed

- defines in the files 
- renamed all of them to conform to ...-cxx14 or ...-libcxx14

thx...

**Comment by ruslo** [comment]
Merged, thanks. Follow-up: https://github.com/ruslo/polly/commit/def6b50d7f08e164ac874232c474ffb3bff9b970

**Pull request merged**